modules/apigee_edge_apidocs/apigee_edge_apidocs.install

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @file
+ * Copyright 2019 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+/**
+ * @file
+ * Install file for Apigee Edge API Docs.
+ */
+
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_requirements().
+ */
+function apigee_edge_apidocs_requirements($phase) {
+  return [
+    'apigee_edge_apidocs_all' => [
```

Comment:
Probably this check should not run in the "update" phase because this could break deployments. Limit this to "install" and "runtime".

Reply:
There shouldn't be any update hooks added to this module.

Reply:
I did not suggest to add an update hook, I suggested to add a condition that checks `in_array($phase, ['install', update']`.
https://github.com/drupal/core/blob/8.6.x/lib/Drupal/Core/Extension/module.api.php#L888-L891

Reply:
I changed the update phase to `REQUIREMENT_WARNING`.